### DISCUSSION TITLE
Congrats! Your request for @yourname was approved and your benefits will be available soon

### DISCUSSION THREAD

**Discussion by aadesh0706** [community/community#140513] [GitHub Education]
### Hi everyone, #140258 @samus-aran #137878 #138447 

![image](https://github.com/user-attachments/assets/b1d9b475-c0bc-41bb-aa82-dde975dcde57)

I was waiting from the last 2 days and still i didnt get my Student Benefits are not approved . Before this it was already got disapproved but now as it is approved i didn't get my Student Benefits .

**Comment by davevad93** [comment]
Hi @aadesh0706 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will change to:

- `Congrats! Your request for @aadesh0706 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
